fades/parsing.py

```diff
@@ -232,12 +233,29 @@ def parse_manual(dependencies):
     return _parse_requirement(dependencies)
 
 
+def _read_lines(filepath):
+    """Read a req file to a list to support nested requirement files."""
+    lines = []
+    with open(filepath, 'rt', encoding='utf8') as fh:
+        for line in fh:
+            line = line.strip()
+            if line.startswith("-r"):
+                logger.debug(
+                    "Reading deps from nested requirement file: %s", line)
+                nested_filename = line.split()[1]
+                nested_filepath = os.path.join(
+                    os.path.dirname(filepath), nested_filename)
+                lines.extend(_read_lines(nested_filepath))
+                continue
+            lines.append(line)
+    return lines
```

Comment:
what about directly `yield` the lines? AFAIU, you could use `yield from` instead of the lines.extend

Reply:
good idea! thanks. (updated)